examples/pytorch/pointcloud/pct/README.md

```diff
@@ -0,0 +1,27 @@
+PCT
+====
+
+This is a reproduction of the paper: [PCT: Point cloud transformer](http://arxiv.org/abs/2012.09688).
+
+# Performance
```

Comment:
Please specify the difference from the paper description, and also include the results from an exact reproduction.

Reply:
updated